src/main/java/org/hibernate/infra/bot/config/RepositoryConfig.java

```diff
@@ -9,6 +9,8 @@ public class RepositoryConfig {
 
 	public JiraConfig jira;
 
+	public Boolean enableEditing;
```

Comment:
I'd move this to `JiraConfig` and call this `insertLinksInPullRequests`... WDYT?

Reply:
Sounds good to me, I'll move and rename the property.